.github/CODEOWNERS

```diff
@@ -0,0 +1,33 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+*       @dl-containers
+
+# TF tests and builds
+tensorflow/ @aws-tensorflow
+tensorflow_serving/ @aws-tensorflow
+testTF* @aws-tensorflow
+testTensor* @aws-tensorflow
+
+# PT tests and builds
+pytorch/ @aws-pytorch
+pytorch_tests/ @aws-pytorch
+
+# MXNet tests and builds
+mxnet/
```

Comment:
are we waiting for the respective aws teams to be created for this?

Reply:
teams created